pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -402,12 +403,16 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 					continue
 				}
 			}
-			r.status.SetDegraded(statusmanager.OperatorConfig, "ApplyOperatorConfig",
-				fmt.Sprintf("Error while updating operator configuration: %v", err))
-			return reconcile.Result{}, err
+			setDegraded = true
 		}
 	}
 
+	if setDegraded {
+		r.status.SetDegraded(statusmanager.OperatorConfig, "ApplyOperatorConfig",
+			fmt.Sprintf("Error while updating operator configuration: %v", err))
```

Comment:
aren't we going to need a way to store the error above, and then use it here? e.g. if the error is cleared when given a set of net-attach-defs `[i, j, k]` where `j` fails, but `k` completes, the error is nil when it gets past the loop and then processes this block for `setDegraded` is true?

Reply:
You're entirely right, great catch!

Reply:
For what it's worth, there's also the scenario where both `i` and `j` fail and cause degraded error. In this case I think we should be okay with returning the last error that occurs (`j`) and once it's resolved the error for `i` will appear.

Reply:
I think last error is probably fine! Thanks!